_build_cfg.yml

```diff
@@ -332,6 +332,8 @@ Dir: dev_guide
 Topics:
   - Name: Overview
     File: index
+  - Name: The Application Life Cycle
```

Comment:
@adellape I see what you're saying. There's not just the one example here... I'll change to "Application Life Cycle Examples".